### ISSUE TITLE
exclude [root]/test folder by default

### ISSUE THREAD

**Issue by timomayer** [biotope/biotope-quality-gate#74]
default lints [root]/test folder which causes a lot of errors, test folder should not be linted in default config.

**Comment by tiagomapmarques** [comment]
I disagree on this point. By default, quality-gate lints everything it finds under `./`. No `src`, `app` or `test` folders are specified - i.e. it does not assume the architecture of the project.
Also, by default, the test files are ignored (i.e. the files ending on `.spec.(ts|js|tsx|jsx)`). Maybe the files that are being linted under your `./test` folder do not follow this convention. If so, then the CLI also provides `--pattern` (or `-p`) so you can specify folder to lint.
For example: `biotope-quality-gate -p src` will ignore the `./test` folder.
